docs/make.jl

```diff
@@ -4,6 +4,8 @@ using PeriodicOrbits
 
 pages = [
     "index.md",
+    "algorithms.md",
```

Comment:
It's good that you added docs, but in the future we should keep PRs separate: each PR is one particular feature/algorithm/fix.

Reply:
We haven't added docs yet to PeriodicOrbits.jl so we could have started this in another PR.

Reply:
Having stuff separated makes PRs easier to review.